tests/dags/providers/provider_api_scripts/test_flickr.py

```diff
@@ -117,6 +62,13 @@ def test_get_media_type():
     assert flickr.get_media_type({}) == "image"
 
 
+def test_get_record_count_from_response():
+    response_json = _get_resource_json("flickr_example_pretty.json")
+    count = flickr.get_record_count_from_response(response_json)
+
+    assert count == "30"
```

Comment:
Perfect example of why this should maybe return an int itself!

Reply:
💯 Should be fixed!